apps/explorer/src/app/components/asset-balance/asset-balance.tsx

```diff
@@ -0,0 +1,32 @@
+import { addDecimalsFormatNumber } from '@vegaprotocol/react-helpers';
+import { AssetLink } from '../links';
+import { useExplorerAssetQuery } from '../links/asset-link/__generated__/Asset';
+
+export type AssetBalanceProps = {
+  assetId: string;
+  price: string;
+};
+
+/**
+ * Given a market ID and a price it will fetch the market
+ * and format the price in that market's decimal places.
+ */
+const AssetBalance = ({ assetId, price }: AssetBalanceProps) => {
+  const { data } = useExplorerAssetQuery({
+    variables: { id: assetId },
+  });
+
+  let label = price;
```

Comment:
nitpick (non-blocking): hmm, would this do the same?
```
const label = addDecimalsFormatNumber(price, data?.asset?.decimals ?? 0)
```
i can't help it but to comment on a `let` declaration when i see it 👮 , even tho it's not really important

Reply:
I have pushed something roughly equivalent, but with fewer `?`s. For me it's a balance between readability and cleverness and squishing it in to one line feels a little needless, but I also don't mind changing it particularly, so I did..